pyomo/contrib/fbbt/interval.py

```diff
@@ -82,9 +106,13 @@ def power(xl, xu, yl, yu):
         if yl >= 0:
             lb = min(xl ** yl, xl ** yu)
             ub = max(xu ** yl, xu ** yu)
+        elif yu <= 0:
+            lb, ub = inv(*power(xl, xu, *sub(0, 0, yl, yu), feasibility_tol), feasibility_tol)
         else:
-            lb = -inf
-            ub = inf
+            lb1, ub1 = power(xl, xu, 0, yu, feasibility_tol)
+            lb2, ub2 = power(xl, xu, yl, 0, feasibility_tol)
+            lb = min(lb1, lb2)
+            ub = max(ub1, ub2)
```

Comment:
Why is the next clause not updated to be relaxed by `feasibility_tol`?

Reply:
Can you be more specific? Which part do you think needs relaxed?

Reply:
Nevermind. I did not see your comment below.